planet/clients/subscriptions.py

```diff
@@ -0,0 +1,171 @@
+"""Planet Subscriptions API Python client."""
```

Comment:
Code of this module was moved here from planet/cli/subscriptions.py with no other changes.